### ISSUE TITLE
Utility of [Remind me] in Subscription management?

### ISSUE THREAD

**Issue by fKunstner** [talks-tue/talks.tue.flask#44]
The utility of the `Remind me` toggle on `/subscription/id` is not clear.

My mental model: 
If I am subscribing, it should remind me. 
If I delete the subscription, it should not. 
Is it an "existing but inactive" subscription state?

**Comment by tim-fi** [comment]
Yes, if you wan't to turn of reminders but still have it listed in your upcoming talks list, I think I missed that part in the introductory meeting...If you activate the account dropdown when logged in, there should be an option there to view all the talks related to your current subscriptions.

**Comment by fKunstner** [comment]
Alright, then a suggestion to make it more obvious;

Instead of having a `/subscription/id` for each subscription, just list them all on the profile and have a delete option and a `Enable email` option.

[Along those lines maybe](https://jsfiddle.net/fw3dgc9L/)

(You do loose the ability to set a different mailing frequency for each list)

**Comment by fKunstner** [comment]
Related issues:
https://github.com/talks-tue/talks.tue.flask/issues/45
https://github.com/talks-tue/talks.tue.flask/issues/48
